.github/workflows/r_nightly.yml

```diff
@@ -77,10 +89,47 @@ jobs:
             echo "No files found. Stopping upload."
             exit 1
           fi
+      - name: Cache Repo
+        uses: actions/cache@v3
+        with:
+          path: repo
+          key: r-nightly-${{ github.run_id }}
+          restore-keys: r-nightly-
+      - name: Sync from Remote

```

Comment:
I think that we don't need this if we use `actions/cache` for `repo/`.
If we use `actions/cache` for `repo/`, `repo/` has the same contents of https://nightlies.apache.org/arrow/r/ eventually.